### PR TITLE
Changes to Repository Name

### PR DESCRIPTION
**Pull request by marcelosousa**

### CODE REVIEW

File: src/models/repository.ts [hunk 1]
```diff
@@ -1,6 +1,23 @@
 export interface Repository {
     id: number;
     name: string;
-    full_name: string;
+    full_name: Name;
     url: string;
+}
+
+interface Name {
+    first_name: string;
```
Comment (Xavier-explore-dev) [inline src/models/repository.ts:9]: This should be called name.

**Review by Xavier-explore-dev — APPROVED** [review]

**Review by Xavier-explore-dev — COMMENTED** [review]
comment

**Review by Xavier-explore-dev — COMMENTED** [review]
comment 2

**Review by Xavier-explore-dev — CHANGES_REQUESTED** [review]
change

**Review by Xavier-explore-dev — APPROVED** [review]

**Review by Xavier-explore-dev — COMMENTED** [review]
qweqwe

**Review by Xavier-explore-dev — CHANGES_REQUESTED** [review]
asdqweqwe

**Review by Xavier-explore-dev — COMMENTED** [review]
qweqwasdasd

**Review by Xavier-explore-dev — APPROVED** [review]

**Review by Xavier-explore-dev — COMMENTED** [review]
qfasdqwe

**Comment by marcelosousa** [comment]
This is a test comment

**Comment by marcelosousa** [comment]
This is a test comment

**Comment by marcelosousa** [comment]
This is a test comment

**Comment by marcelosousa** [comment]
This is a test comment

**Comment by marcelosousa** [comment]
This is a test comment

File: README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+* This is a demo project *
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (marcelosousa) [inline README.md:3]: This is a test comment

File: README.md [hunk 3]
```diff
@@ -21,3 +23,4 @@ If you aren’t satisfied with the build tool and configuration choices, you can
 Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
 
 You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+
```
Comment (marcelosousa) [inline README.md:26]: Pls rm new line

File: README.md (hunk 2 above)
Comment (marcelosousa) [reply README.md:3]: This is a test reply

File: src/components/GitHubController.tsx [hunk 4]
```diff
@@ -11,6 +11,8 @@ export default function GitHubController({ children }: { children: ReactNode })
     const [repoWithMostStars, setRepoWithMostStars] = useState<Repository>();
     const [repoHighlighted, setRepoHighlighted] = useState<Repository>();
 
+    console.log('Say hello');
```
Comment (marcelosousa) [inline src/components/GitHubController.tsx:14]: Change to `say bye`

File: README.md [hunk 5]
```diff
@@ -1,4 +1,7 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+* This is a demo project *
+
```
Comment (marcelosousa) [inline README.md:2]: Test comment

File: README.md [hunk 6]
```diff
@@ -1,4 +1,7 @@
...
+
```
Comment (marcelosousa) [inline README.md:3]: Remove this line

**Comment by marcelosousa** [comment]
Comment test

File: README.md [hunk 7]
```diff
@@ -1,4 +1,7 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+* This is a demo project *
```
Comment (marcelosousa) [inline README.md:1]: Test comment

Comment (marcelosousa) [reply README.md:1]: Reply

File: README.md (hunk 5 above)
Comment (marcelosousa) [inline README.md:2]: Test comment

File: src/components/GitHubController.tsx [hunk 8]
```diff
@@ -11,6 +11,8 @@ export default function GitHubController({ children }: { children: ReactNode })
     const [repoWithMostStars, setRepoWithMostStars] = useState<Repository>();
     const [repoHighlighted, setRepoHighlighted] = useState<Repository>();
```
Comment (ferreiratiago) [inline src/components/GitHubController.tsx:12]: Comment

File: src/components/GitHubController.tsx [hunk 9]
```diff
@@ -11,6 +11,8 @@ export default function GitHubController({ children }: { children: ReactNode })
...
```
Comment (ferreiratiago) [inline src/components/GitHubController.tsx:13]: Comment

File: README.md (hunk 6 above)
Comment (marcelosousa) [inline README.md:3]: Test comment

File: src/models/repository.ts [hunk 10]
```diff
@@ -1,6 +1,23 @@
 export interface Repository {
     id: number;
     name: string;
-    full_name: string;
+    full_name: Name;
```
Comment (marcelosousa) [inline src/models/repository.ts:4]: I don't like this variable name.

File: src/components/RepositoryDetails/RepositoryDetails.tsx [hunk 11]
```diff
@@ -26,7 +26,7 @@ export default React.memo(function RepositoryDetails({ repo }: { repo: Repositor
 
     return (
         <div style={style} onMouseEnter={addHighlight} onMouseLeave={clearHighlight}>
-            <p>Name: {repo.full_name}</p>
+            <p>Name: {repo.full_name.first_name}</p>
```
Comment (marcelosousa) [inline src/components/RepositoryDetails/RepositoryDetails.tsx:29]: Test comment

File: src/components/GitHubController.tsx (hunk 9 above)
Comment (ferreiratiago) [reply src/components/GitHubController.tsx:13]: Reply 1

Comment (ferreiratiago) [reply src/components/GitHubController.tsx:13]: Reply 2

File: src/components/RepositoryDetails/RepositoryDetails.tsx (hunk 11 above)
Comment (marcelosousa) [reply src/components/RepositoryDetails/RepositoryDetails.tsx:29]: Another test comment

File: README.md (hunk 6 above)
Comment (marcelosousa) [reply README.md:3]: Test reply

**Pull request closed without merging**